pkg/kapp/diffgraph/change_rule.go

```diff
@@ -38,7 +38,7 @@ func NewChangeRuleFromAnnString(ann string) (ChangeRule, error) {
 	pieces := strings.Split(ann, " ")
 	if len(pieces) != 4 {
 		return ChangeRule{}, fmt.Errorf(
-			"Expected change rule annotation value to have 4 pieces but was %d", len(pieces))
+			"Change rule annotation value should be of the format: (upsert|delete) (before|after) (upserting|deleting) (groupName)")
```

Comment:
May be instead of groupName if we have ChangeGroupName, will that be more explicit ?

Reply:
Actually, targetGroupName might be a better choice, but I am not sure which is more descriptive.

Reply:
lets tweak it to

```
Expected change rule annotation value to have format: '(upsert|delete) (before|after) (upserting|deleting) (change-group)'
```

Reply:
Done

Reply:
Error message now looks like this
<img width="1019" alt="Screenshot 2021-09-09 at 7 43 33 PM" src="https://user-images.githubusercontent.com/8457124/132702226-7630a769-793f-477d-a808-9350ba73045e.png">

Reply:
Sentence looks incomplete. Suggestions:
Expected change rule annotation value to have format: '(upsert|delete) (before|after) (upserting|deleting) (change-group)', provided 'server'

Reply:
Updated the message to 
```
Expected change rule annotation value to have format: '(upsert|delete) (before|after) (upserting|deleting) (change-group)', but was 'server'
```

@renuy @rohitagg2020 @cppforlife